src/lib/greenboot.rs

```diff
@@ -239,44 +241,51 @@ mod test {
         let _ = std::fs::remove_file(format!("{}/01_failing_binary", required_path));
```

Comment:
We do have tear_down() function at the end of each function which ideally should clear the files ,but again it may be happen iff any test is not yet finished or stays unfinished this test may get affected. 
I believe you are using `sudo cargo test --features test-remount -- --test-threads=1` this command so that the test data does not interfere. 
I tried to run test by commenting this 3 line(240,241,242) tests run fine for me.
 Removing the files at the start of the test as a precaution is good. so we can still keep this but can you remove the files in all the paths instead of GREENBOOT_INSTALL_PATHS[1]. like you have done in other test.

Reply:
Good spot, I believe those lines originate from Armando's PR and I missed editing them in here. I just pushed a fix to incorporate them into the loop with all install paths. Thank you!